### DISCUSSION TITLE
Proposal for programs in rust

### DISCUSSION THREAD

**Discussion by sudormrfbin** [HarshCasper/Rotten-Scripts#838] [Ideas]
With reference to #811. I started learning rust recently and do not have a lot of experience but I definitely think it is a *very* promising and enjoyable language to work with.

## Prior Art
- https://github.com/maralla/pack/pull/54
- https://github.com/maralla/pack/pull/53

## Proposed Programs
- A simple command line music player
- YouTube video downloader

cc @HarshCasper

**Comment by vybhav72954** [comment]
@HarshCasper @seema1711 @sohamsshah Can you guys have a look at this

**Reply by iseemasaharan** [reply]
I think it's a good idea.

**Comment by sudormrfbin** [comment]
Can I go ahead with this idea or is something more necessary ?

**Reply by vybhav72954** [reply]
Just waiting for PA's review. I will personally get in touch about this.

**Comment by HarshCasper** [comment]
Hi @gokulsoumya 

Please go forward with this: Create Issues and take them up with `/assign` and submit PRs :sparkles: 

I have developed some expertise in Rust so I will be able to review your PRs :)

**Reply by vybhav72954** [reply]
I can help as well, although am a noob as of now.
